presto-docs/src/main/sphinx/admin/resource-groups.rst

```diff
@@ -9,287 +9,458 @@ Except for the limit on queued queries, when a resource group runs out of a reso
+while database-based resource management is more complex to set up but offers more flexibility and dynamic changes.
+
+
+File Resource Group Manager
+---------------------------
+
+The File Resource Group Manager in PrestoDB is a way to manage resources using a JSON configuration file.
+This file contains definitions of all resource groups and rules for selecting the appropriate resource group
+for a given query.
+
+Here's a brief guide on how to set it up:
+
+Create a JSON file: This file should contain the definitions for your resource groups.
+Each resource group can specify things like the maximum memory, the maximum queued queries,
+and the maximum running queries.
+
+Specify the file in the Presto configuration: In the etc directory of your Presto installation,
+create a file named ``resource-groups.properties``. In this file, add a line that specifies the location of
+your JSON file: ``resource-groups.config-file``=<file_path>, where <file_path> is the path to your JSON file.
+
+Restart the Presto server: After creating the JSON file and pointing to it in the Presto configuration,
+you need to restart your Presto server. The new resource groups will take effect after the restart.
+
+.. code-block:: text
 
     resource-groups.configuration-manager=file
-    resource-groups.config-file=etc/resource_groups.json
+    resource-groups.config-file=etc/resource-groups.json
+
+
+Database Resource Group Manager
+-------------------------------
+The Database Resource Group Manager in PrestoDB is a way to manage resources using a relational database.
+This allows dynamic changes to the resource group configurations without needing to restart
+the Presto server.
+
+Here's a brief guide on how to set it up:
+
+Set up a relational database: The database should be accessible by Presto.
+It will be used to store the configurations of your resource groups.
+
+Create tables for resource groups and selection rules: You need to create tables in the
+database that will store the definitions of your resource groups and the rules for selecting
+the appropriate resource group for a given query.
+
+Specify the database in the Presto configuration: In the etc directory of your Presto installation,
+create a file named ``resource-groups.properties``. In this file, add a line that specifies
+the JDBC URL of your database: ``resource-groups.config-db-url``=<jdbc_url>, where <jdbc_url> is the
+JDBC URL of your database.
+
+The database resource group manager loads the configuration from a relational database. The
+supported databases are MySQL, PostgreSQL, and Oracle.
+
+.. code-block:: text
+
+    resource-groups.configuration-manager=db
+    resource-groups.config-db-url=jdbc:mysql://localhost:3306/resource_groups
+    resource-groups.config-db-user=username
+    resource-groups.config-db-password=password
+
+Here's an example of what the tables in your database might look like:
+
+``Resource Groups Table``
```

Comment:
Consider formatting _Resource Groups Table_ and _Selectors Table_ as sub-level headers. When these are formatted like the items in them (resource_group_id, soft_memory_limit), they look like part of the list of items and not a title for the list of items. 

<img width="750" alt="Screenshot 2023-09-05 at 3 04 13 PM" src="https://github.com/prestodb/presto/assets/7013443/ac72b9fa-7c25-4c4a-83ad-e1ad0750f1b7">